wordpress-mobile/WordPress-Android
# QR Code: Partial MLKit Revert

Fixes #19695 

This PR reverts the use of MLKit for scanning QR Code as used by the web login. The reason for the revert is because the `stories-android` has a dependency on camerax `1.0.0-alpha06' and the MLKit needs at least version '1.2.3'. We could not resolve the dependency via gradle :(.

The entire `barcodescanner` package had to be removed because the dependencies were nixed from build.gradle. Since the strings were already translated, I marked the resource as unused so they would not have to go through translation again.

The plan is to update the `stories-android` to CameraX version `1.2.3` during the next hack week

cc: @oguzkocer 

## To Test:
Story Post
- Install the JP app build from this PR
- Navigate to My Site
- Tap on the FAB and select Story Post
- Navigate past the intro (if shown) and agree to any permissions
- Tap the "camera" icon
- ✅ Verify the camera launches and the app doesn't crash

[JP] Scan Login Code - Success
- Log in to the Jetpack app with a WP.com account (note that you need to use a non A8C account and a non 2FA enabled account).
- Navigate to the Me screen (click on bottom Me navigation tab).
(STOP)
- Head over to your desktop and open a web browser (note that using an incognito tab works best).
- Browse to wordpress.com (note that if you are logged-in, log-out first).
- Tap the Log In link (top-right).
- Tap the Login via the mobile app link in the list of options below the main Continue button (bottom-middle).
- Verify you are on the Login via the mobile app view and Use QR Code to login is shown, along with a QR code for you to scan.

## Regression Notes

1. Potential unintended areas of impact
Stories still crash when launching camera

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist: N/A
